tests/unit/type/stream-cgroups.tcl

```diff
@@ -482,22 +482,22 @@ start_server {
         r XDEL x 103
 
         set reply [r XINFO STREAM x FULL]
-        assert_equal [llength $reply] 12
+        assert_equal [llength $reply] 16
         assert_equal [lindex $reply 1] 4 ;# stream length
-        assert_equal [lindex $reply 9] "{100-0 {a 1}} {101-0 {b 1}} {102-0 {c 1}} {104-0 {f 1}}" ;# entries
-        assert_equal [lindex $reply 11 0 1] "g1" ;# first group name
-        assert_equal [lindex $reply 11 0 7 0 0] "100-0" ;# first entry in group's PEL
-        assert_equal [lindex $reply 11 0 9 0 1] "Alice" ;# first consumer
-        assert_equal [lindex $reply 11 0 9 0 7 0 0] "100-0" ;# first entry in first consumer's PEL
-        assert_equal [lindex $reply 11 1 1] "g2" ;# second group name
-        assert_equal [lindex $reply 11 1 9 0 1] "Charlie" ;# first consumer
-        assert_equal [lindex $reply 11 1 9 0 7 0 0] "100-0" ;# first entry in first consumer's PEL
-        assert_equal [lindex $reply 11 1 9 0 7 1 0] "101-0" ;# second entry in first consumer's PEL
+        assert_equal [lindex $reply 13] "{100-0 {a 1}} {101-0 {b 1}} {102-0 {c 1}} {104-0 {f 1}}" ;# entries
+        assert_equal [lindex $reply 15 0 1] "g1" ;# first group name
+        assert_equal [lindex $reply 15 0 11 0 0] "100-0" ;# first entry in group's PEL
+        assert_equal [lindex $reply 15 0 13 0 1] "Alice" ;# first consumer
+        assert_equal [lindex $reply 15 0 13 0 7 0 0] "100-0" ;# first entry in first consumer's PEL
+        assert_equal [lindex $reply 15 1 1] "g2" ;# second group name
+        assert_equal [lindex $reply 15 1 11 0 1] "Charlie" ;# first consumer
+        assert_equal [lindex $reply 15 1 13 0 7 0 0] "100-0" ;# first entry in first consumer's PEL
+        assert_equal [lindex $reply 15 1 13 0 7 1 0] "101-0" ;# second entry in first consumer's PEL
```

Comment:
why did you have to change these indexes? is it because the reply is a map and the test ignores it and is coupled with a specific indexes?
lets see of we can improve.

Reply:
Are you talking about RESP3 tests? :)

Sure, that was the hardest part of this PR (aligning the indices)... Oh TCL wizard, please provide guidance for improvement.

Reply:
yes, we can do one of two things (which i think we must apply to all these tests that mess with XINFO output and hard coded indexes).
1. use `HELLO 3` and then we can process a map. the downside is that we may want to change protocol before and after each XINFO call, so that the rest of the test is in RESP2 (although i don't think it's a must).
2. pass the return value of XINFO though some zipping function that will translate it to a dict

2 is basically this:
```tcl
set l {a 1 b 2 c 3}
set d [dict create {*}$l]
assert_equal [dict get $d b] 2
```

Reply:
Resolved via 72a101513, without having to convert the list as TCL allows it :)